tests/libvirt.Tests/CommonTests.cs

```diff
@@ -0,0 +1,13 @@
+using Xunit;
+
+namespace libvirt.Tests
+{
+    public class CommonTests
+    {
+        [Fact]
+        public void GetVersion_ReturnVersion()
+        {
+            Libvirt.GetVersion();
```

Comment:
Assert is missing

Reply:
I have no idea how to assert because the version is provided by the dll or so, The way of test fail should be a unhandled exception?

Reply:
Maybe you can assert that version is > 0.0.1? I mean, a valid version? If there is valid test, we can annotate the code with ExcludeFromCodeCoverageAttribute